.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - restore_cache:
           keys:
             - m2-{{ checksum "build.sbt" }}
-            - m2
+            - daml-{{ checksum "project/Dependencies.scala" }}-{{ checksum "src/test/resources/ping-pong/daml.yaml" }}
```

Comment:
built.sbt should be part of the key as well

Reply:
:+1: